core/src/main/resources/lib/layout/breadcrumbs.css

```diff
@@ -109,7 +109,7 @@
 
 #menuSelector {/* used for showing 'v' on the right of the anchor */
     background-color:transparent;
-    background-image: url(menu_down_arrow.png);
+    background-image: var(--menu-selector-down-background-image);
```

Comment:
anyone able to check if this works on IE 11?

![image](https://user-images.githubusercontent.com/21194782/83039862-8a77b300-a036-11ea-8bd1-1a3da4e02d57.png)


I suspect that it needs a manual fallback, looks like the postcss processor isn't adding the fallback for it

Reply:
Seems to work on i.e but the entire dark theme doesn't

Here is a side by side with ie11 on the left and edge on the right

![image](https://user-images.githubusercontent.com/31362124/83043762-d9910880-a075-11ea-91c7-a2bec1e7e6cd.png)

Reply:
> Seems to work on i.e but the entire dark theme doesn't

It's not expected to work on IE 11 at all for the theme

> Here is a side by side with ie11 on the left and edge on the right

That's not working, at least for fonts, the preprocessor isn't picking them up 😢,

Two options I guess,

1. this work doesn't really require the fonts to be changeable at runtime, so it could go back to less variables
2. try out the other postcss plugin and see if that handles fonts, this one only seems to handle colours

cc @fqueiruga

Reply:
Some more observations I made:
- The administrative monitor (The bell thing in the top right) still shows white background on the dark theme
- CLI page has black text with dark theme.

Reply:
Thanks I’d forgotten about the admin monitor issue, and I hadn’t checked the CLI page

Reply:
BTW I would not bother converting the arrow to a CSS variables. It should be either an icon or a CSS triangle, not an image anyway.

Reply:
https://issues.jenkins-ci.org/browse/JENKINS-62496